toolchain/check/facet_type.cpp

```diff
@@ -326,14 +326,20 @@ static auto TryGetImplWitnessAccessOfPeriodSelf(Context& context,
 // `ImplWitnessAccess` will not loop infinitely.
 class SubstImplWitnessAccessCallbacks : public SubstInstCallbacks {
  public:
+  // The `facet_type_info` is for the facet whose rewrite constraints are being
+  // substituted, and where it looks for rewritten values to substitute from.
+  //
+  // The `substituting_constaint` is the rewrite constraint for which the RHS is
+  // being substituted with the value from another rewrite constraint, if
+  // possible. That is, `.Y = .X` in the example in the class docs.
   explicit SubstImplWitnessAccessCallbacks(
       Context* context, SemIR::LocId loc_id,
-      const SemIR::FacetTypeInfo* facet_type,
-      const SemIR::FacetTypeInfo::RewriteConstraint* current_constraint)
+      const SemIR::FacetTypeInfo* facet_type_info,
+      const SemIR::FacetTypeInfo::RewriteConstraint* substituting_constaint)
```

Comment:
```suggestion
      const SemIR::FacetTypeInfo::RewriteConstraint* substituting_constraint)
```

(and likewise for the other occurrences of `substituting_constaint`)

Reply:
Oops, thanks, done